source/includes/APIReference/CustomFields/_create.md.erb

```diff
@@ -0,0 +1,40 @@
+## Create Custom Fields
+
+<%= partial "includes/APIReference/CustomFields/Examples/create.tmpl.erb" %>
+
+Add one or more customfields.
+
+### HTTP Request
+
+<img src="images/post.png" alt="post"/><api>https://rest.tsheets.com/api/v1/customfields</api>
+
+### Properties
+_Pass an array of customfield objects as the value to a 'data' property (see example)._
+
+|                |             |             |
+| -------------: | :---------: | ----------- |
+| **name**<br/>required | _String_ | The name of this customfield. |
+| **required**<br/>optional | _Boolean_ | If true, a non-null value must be selected for this customfield when it appears on a timesheet. |
+| **active**<br/>optional | _Boolean_ | If false, this `customfield` is considered archived and will not be visible or usable. |
+| **applies_to**<br/>required | _String_ | Allowed values: 'timesheet'. Specify that this `customfield` should appear on timesheets (additional customfield types may be supported in the future). |
+| **type**<br/>required | _String_ | Allowed values: 'managed-list', 'free-form'.  A managed-list customfield is intended to be displayed as a dropdown list, where each 'option' is a customfielditem. A 'free-form' customfield supports text entry and does not require a collection of customfielditems to accompany it. |
+| **show_to_all**<br/>optional | _Boolean_ | Default: `true`.  Declares whether this customfield should exist on timesheets for all workers. If false, it will only appear on timesheets for workers to whom it has been assigned. |
```

Comment:
We don't support filtering/assignment based on workers for _fields_, only for items.  For fields, this is only controlled based on jobcode.  This 'show_to_all' setting corresponds to the 'Show for all customers' setting in the UI.  Hmm... perhaps we should rename this to 'show_for_all_jobcodes' ?

So apart from the name of the property, I suggest we change this to:

```suggestion
| **show_to_all**<br/>optional | _Boolean_ | Default: `true`.  Declares whether this customfield should be shown on timesheets regardless of the jobcode chosen. If false, it will only appear when the chosen jobcode for a timesheet has been associated with this field. |
```

Reply:
Second this comment, the filtering applies to job codes, not workers.